component-models.json

```diff
@@ -312,12 +312,79 @@
         "label": "Image",
         "multi": false
       },
+      {
+        "component": "text-input",
+        "valueType": "string",
+        "name": "title",
+        "value": "",
+        "label": "Title"
+      },
+      {
+        "component": "select",
+        "name": "type",
+        "value": "h2",
+        "label": "Type",
+        "valueType": "string",
+        "options": [
+          {
+            "name": "h1",
+            "value": "h1"
+          },
+          {
+            "name": "h2 - Default",
+            "value": "h2"
+          },
+          {
+            "name": "h3",
+            "value": "h3"
+          },
+          {
+            "name": "h4",
+            "value": "h4"
+          },
+          {
+            "name": "h5",
+            "value": "h5"
+          },
+          {
+            "name": "h6",
+            "value": "h6"
+          }
+        ]
+      },
+      {
+        "component": "text-input",
+        "valueType": "string",
+        "name": "subTitle",
+        "value": "",
+        "label": "Sub Title"
+      },
       {
         "component": "text-area",
         "name": "text",
         "value": "",
```

Comment:
@mayur-satav could you please update the name and label to accurately reflect the purpose of the field? Perhaps using terms like 'description' or 'summary' would be more descriptive.

Reply:
ok